osu.Game/Online/Chat/MessageNotifier.cs

```diff
@@ -127,6 +127,8 @@ private bool checkForPMs(Channel channel, Message message)
             if (!notifyOnPrivateMessage.Value || channel.Type != ChannelType.PM)
                 return false;
 
+            host.Window?.Flash();
```

Comment:
I think flashing the taskbar should be a feature accessible to all notifications. A property of notifications, something like `bool FlashTaskbar { get; init; }`. Flashing when a beatmap import (from stable, on a slow connection, etc.) is finished would also be useful. The default could even be `true` if it makes sense.

Also, the taskbar should probably only be flashed if it doesn't have focus (`!window.IsActive.Value`).

Reply:
> I think flashing the taskbar should be a feature accessible to all notifications. A property of notifications, something like bool FlashTaskbar { get; init; }. Flashing when a beatmap import (from stable, on a slow connection, etc.) is finished would also be useful. The default could even be true if it makes sense.

Is the `IsImportant` property could be of any use here? I feel like we could use it to change the flash behaviour too.

> Also, the taskbar should probably only be flashed if it doesn't have focus (!window.IsActive.Value).

Framework already guards this case

https://github.com/ppy/osu-framework/blob/d8052f2ad21748c80eb119f6038e576ef943e896/osu.Framework/Platform/SDL2Window.cs#L410-L414

Reply:
I think flashing for `IsImportant` makes most sense.